context/app/static/js/shared-styles/sections/TableOfContents/hooks.ts

```diff
@@ -52,4 +56,13 @@ function useFindActiveIndex({
   }, [currentSection, setCurrentSection, clickedRef, itemsWithNodeRef]);
 }
 
-export { useThrottledOnScroll, useFindActiveIndex };
+function useAnimatedSidebarPosition() {
+  const { summaryInView } = useEntityStore((state: EntityStore) => state.summaryComponentObserver);
+  const initialHeightOffset = headerHeight + 16;
```

Comment:
What is the `16` referring to here - where does this unit come from?

Reply:
This was adding 1rem of padding to the top of the sidebar to keep it from touching the entity header; now that John's entity header work is merged in, I was able to replace this.